src/client.rs

```diff
@@ -11,27 +11,27 @@ use std::time::Duration;
 
 use cadence::{prelude::*, StatsdClient};
 use error_chain::ChainedError;
-use futures::future;
 use futures::future::Either;
 use futures::sync::mpsc;
 use futures::sync::oneshot::Receiver;
 use futures::AsyncSink;
+use futures::{future, try_ready};
 use futures::{Async, Future, Poll, Sink, Stream};
-use reqwest::async::Client as AsyncClient;
+use reqwest::r#async::Client as AsyncClient;
```

Comment:
What's r# mean?

Reply:
It's the [raw identifier syntax](https://rust-lang-nursery.github.io/edition-guide/rust-2018/module-system/raw-identifiers.html).